internal/webserver/handler-ui.go

```diff
@@ -16,185 +14,8 @@ import (
-			// Convert `src` attributes
-			if src != "" {
-				archivalName := getArchivalName(src)
-				if archivalName != "" && archive.HasResource(archivalName) {
-					node.SetAttr("src", createArchivalURL(archivalName))
-				}
-			}
-
-			// Split srcset by comma, then process it like any URLs
-			srcSets := strings.Split(strSrcSets, ",")
-			for i, srcSet := range srcSets {
-				srcSet = strings.TrimSpace(srcSet)
-				parts := strings.SplitN(srcSet, " ", 2)
-				if parts[0] == "" {
-					continue
-				}
-
-				archivalName := getArchivalName(parts[0])
-				if archivalName != "" && archive.HasResource(archivalName) {
-					archivalURL := createArchivalURL(archivalName)
-					srcSets[i] = strings.Replace(srcSets[i], parts[0], archivalURL, 1)
-				}
-			}
-
-			if len(srcSets) > 0 {
-				node.SetAttr("srcset", strings.Join(srcSets, ","))
-			}
-		})
-
-		bookmark.HTML, err = goquery.OuterHtml(doc.Selection)
-		checkError(err)
-	}
-
-	// Execute template
-	if developmentMode {
-		if err := h.PrepareTemplates(); err != nil {
-			log.Printf("error during template preparation: %s", err)
-		}
-	}
-
-	tplData := struct {
-		RootPath string
-		Book     model.Bookmark
-	}{h.RootPath, bookmark}
-
-	err = h.templates["content"].Execute(w, &tplData)
-	checkError(err)
-}
-
-// ServeThumbnailImage is handler for GET /bookmark/:id/thumb
-func (h *Handler) ServeThumbnailImage(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
-	// Get bookmark ID from URL
-	ctx := r.Context()
-
-	// Get bookmark ID from URL
-	strID := ps.ByName("id")
-
-	// Get bookmark from database
-	id, err := strconv.Atoi(strID)
-	checkError(err)
-	bookmark, exist, err := h.DB.GetBookmark(ctx, id, "")
-	checkError(err)
-
-	if !exist {
-		log.Println("error: bookmark not found")
-		return
-	}
-
-	// If it's not public, make sure session still valid
-	if bookmark.Public != 1 {
-		err = h.validateSession(r)
-		if err != nil {
-			newPath := path.Join(h.RootPath, "/login")
-			redirectURL := createRedirectURL(newPath, r.URL.String())
-			redirectPage(w, r, redirectURL)
-			return
-		}
-	}
-	// Open image
-	imgPath := fp.Join(h.DataDir, "thumb", strID)
-	img, err := os.Open(imgPath)
-	checkError(err)
-	defer img.Close()
-
-	// Get image type from its 512 first bytes
-	buffer := make([]byte, 512)
-	_, err = img.Read(buffer)
-	checkError(err)
-
-	mimeType := http.DetectContentType(buffer)
-	w.Header().Set("Content-Type", mimeType)
-
-	// Set cache value
-	info, err := img.Stat()
-	checkError(err)
-
-	etag := fmt.Sprintf(`W/"%x-%x"`, info.ModTime().Unix(), info.Size())
-	w.Header().Set("ETag", etag)
-	w.Header().Set("Cache-Control", "max-age=86400")
-
-	// Serve image
-	if _, err := img.Seek(0, 0); err != nil {
-		log.Printf("error during image seek: %s", err)
-	}
-	_, err = io.Copy(w, img)
-	checkError(err)
-}
-
 // ServeBookmarkArchive is handler for GET /bookmark/:id/archive/*filepath
 func (h *Handler) ServeBookmarkArchive(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
```

Comment:
You are right, I have to cleanup this methods. Since I switched them to public methods I have to manually find and delete them because the linters does not show non-usage warning for them.